Gemfile

```diff
@@ -4,9 +4,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in importmap-rails.gemspec.
 gemspec
 
-rails_version = ENV["RAILS_VERSION"] || "6.1.0"
-gem "rails", "~> #{rails_version}"
-
+gem "appraisal"
+gem "rails", "~> 6.0.0"
+gem "sprockets-rails"
```

Comment:
This file defaulted the Rails version to 6.1, but the [gemspec specifies 6.0](https://github.com/rails/cssbundling-rails/blob/b27be4a4226368f13be515013c8120b1b920d072/cssbundling-rails.gemspec#L14).  Which is correct?  (Or do we want to jump to Rails 7.0?)